### ISSUE TITLE
Not working with Wintermute tld.exe (2.05)

### ISSUE THREAD

**Issue by atomice1** [FINDarkside/TLD-Developer-Console#27]
tld.exe has now been forked so that Wintermute now uses a separate executable in tld_dlc\wintermute\tld.exe, however the mod is not working with this executable using MelonLoader v0.6.1, mod version 1.5.1 (also tried 1.5.0 and 1.4.0). The version of tld_dlc\wintermute\tld.exe is 2.05 (NOT 2.06 like the main executable).

**Comment by atarugolan** [comment]
Yes I confirm, also the MelonLoader don't working, so i think isn't the console not working but is the MelonLoader

edit: I try with one version of Wintermute run separatly so i can use the Melonloader, and the cosole start with asking dependency, so i add the "missing dependency" so the Mod load succesfully but in real not work. please if you want much infos i can give to you, please help to solve that also in wintermute

**Comment by yuk75** [comment]
Still not working for wintermute :(
